modules/nf-core/qualimap/bamqccram/main.nf

```diff
@@ -57,4 +57,15 @@ process QUALIMAP_BAMQCCRAM {
         samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
     END_VERSIONS
     """
+
+    stub:
+    prefix = task.ext.suffix ? "${meta.id}${task.ext.suffix}" : "${meta.id}"
```

Comment:
Is there no separator here? And why does it need suffix?

Reply:
what about the no separator?

For the second question, I just copied the line setting up the prefix.
I didn't looked about why it's doing that.

Reply:
scratch that, I have no idea where this is coming from, bad copy paste I'd say, I'll fix it right away